pom.xml

```diff
@@ -41,7 +41,7 @@
   <description>
     This plugin provides an stable API to Bouncy Castle related tasks.
   </description>
-  <url>https://github.com/jenkinsci/bouncycastle-api-plugin</url>
+  <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
```

Comment:
Why does incrementals need this - it is less readable and much worse IMO.

Reply:
> Why does incrementals need this

@jtnord Please the following links:

- [JENKINS-58716](https://issues.jenkins.io/browse/JENKINS-58716)
- https://github.com/jenkinsci/archetypes/pull/192
- https://github.com/jenkinsci/incrementals-tools/pull/16
- https://github.com/jenkinsci/plugin-compat-tester/pull/181
- https://github.com/jenkins-infra/community-functions/pull/23

Reply:
very late to the party - but github (which is all we support for jenkins plugins IIUC) can checkout the sha of a commit from a fork so long as there was a PR - so this should all be unnecessary - just was missing the correct FETCH options?

pushing incrementals from non PRs is not supported?

Reply:
I don't know, James, but this seems like the wrong forum to debate a change that we have already rolled out to dozens of other plugins.